addons/sourcemod/scripting/freak_fortress_2.sp

```diff
@@ -2178,30 +2185,44 @@ stock bool:CheckSoundFlags(client, soundFlags)
 
 	if(IsFakeClient(client))
 	{
-		return true;
+		return false;
 	}
 
 	if(muteSound[client] & soundFlags)
 	{
-		return false;
+		return true;
```

Comment:
this should be kept as return false

Reply:
These sound functions are all using misleading names when you created them. I don't want to change their implementation, only their names.